packages/form-js-editor/src/render/components/FormEditor.js

```diff
@@ -270,9 +270,7 @@ export default function FormEditor(props) {
     getService(type, strict = true) {
 
       // TODO(philippfromme): clean up
-      if (type === 'formFieldRegistry') {
-        return new Map();
```

Comment:
I searched the entire codebase and found that it simply is never called. 

There is however a registered version of `formFieldRegistry` in the editor's DI. My best guess is that it was once used for mocking, but since all usages of FormFieldRegistry in the editor package directly DI instead of using 'useService', this has never popped up. So I found a bug in advance basically.